css/yzecoriolis.css

```diff
@@ -1751,6 +1751,9 @@
   color: white;
   transform: scale(1.05);
 }
+.yzecoriolis.sheet.actor .sheet-body .critical-injuries-list .crit-control-widget.item-delete {
+  padding-left: 15px;
```

Comment:
Added a bit of padding-left to the delete button to help reduce accidental deletion of items.